powa--3.2.1dev.sql

```diff
@@ -1235,6 +1237,7 @@ BEGIN
         JOIN pg_roles r ON pgss.userid = r.oid
         WHERE pgss.query !~* ignore_regexp
         AND NOT (r.rolname = ANY (string_to_array(current_setting('powa.ignored_users'),',')))
+        AND dbid IN (SELECT oid FROM powa_databases WHERE dropped IS NULL)
```

Comment:
because i think it is more readable this way… and joining for an existence test is usually not the best way. I could have written an exists test instead, but that would be the same